packages/form/src/SelectList.js

```diff
@@ -65,29 +74,48 @@ class SelectList extends PureComponent {
         multiple: false,
         showCheckAll: true,
         minCheck: 0,
-        allOptionLabel: 'All',
-        values: undefined,
-        defaultValues: [],
+        checkAllLabel: 'All',
+        value: undefined,
+        defaultValue: undefined,
         onChange: () => {},
         title: undefined,
         desc: undefined,
     };
 
     state = {
-        // eslint-disable-next-line react/destructuring-assignment
-        checkedState: getInitialCheckedState(this.props.values || this.props.defaultValues),
+        checkedState: getInitialCheckedState(
+            this.getInitialValue(),
+            this.props.multiple
+        ),
     };
 
     componentWillReceiveProps(nextProps) {
+        warning(
+            this.getIsControlled(this.props) === this.getIsControlled(nextProps),
```

Comment:
Should this be `!==`?

```suggestion
            this.getIsControlled(this.props) !== this.getIsControlled(nextProps),
```

Reply:
The first parameter of `warning()` should be "correct case".  The function will popup warning if this value is falsy.

ref: https://github.com/BerkeleyTrue/warning#usage